tools/pylint-extensions/pylint-guidelines-checker/tests/test_pylint_custom_plugins.py

```diff
@@ -3136,4 +3136,84 @@ def test_raise_traceback(self):
                     end_col_offset=96
                 )
         ):
-            self.checker.visit_importfrom(node)
\ No newline at end of file
+            self.checker.visit_importfrom(node)
+
+class TestTypePropertyNameLength(pylint.testutils.CheckerTestCase):
```

Comment:
can we add two more tests:

test_instance_attr_name_too_long and test_class_var_name_too_long?